src/Menu.ts

```diff
@@ -44,7 +44,15 @@ export default function makeMenu(spec: Spec) {
 
   const menuToggle = spec.doc.createElement('div');
   menuToggle.setAttribute('id', 'menu-toggle');
-  menuToggle.textContent = '☰';
+  menuToggle.innerHTML =
+    // a square "hamburger" menu symbol consisting of three horizontal lines,
+    // similar in appearance to U+2630 TRIGRAM FOR HEAVEN ☰
+    `<svg xmlns="http://www.w3.org/2000/svg"
+        stroke="black" style="width:100%; height:100%; stroke:currentColor"
```

Comment:
```suggestion
        style="width:100%; height:100%; stroke:currentColor"
```

`stroke` is specified twice here - I don't _think_ that does anything, though I am not confident in my SVG knowledge.

Reply:
It's a _very_ old habit from when currentColor support was mixed; I wanted to make sure that the fallback would be black lines rather than transparent ones. But [caniuse](https://caniuse.com/?search=currentcolor) makes clear that such precautions are unnecessary now.